package.json

```diff
@@ -0,0 +1,68 @@
+{
+    "name": "@comparaonline/openapi-generator",
+    "version": "1.4.6",
+    "description": "Soap and rest connector",
+    "main": "build/index.js",
+    "types": "build/index.d.ts",
+    "repository": "https://github.com/comparaonline/mono-client.git",
```

Comment:
This is ok? same in description

Reply:
It's true, now I do it correctly.
Thanks